intelmq/bots/parsers/netlab_360/parser.py

```diff
@@ -21,18 +22,28 @@ def parse_line(self, line, report):
 
             if report['feed.url'] in Netlab360ParserBot.DGA_FEED:
                 event.add('source.fqdn', value[1])
-                event.add('time.source', value[3] + ' UTC')
+                # DGA Feed format is
+                # DGA family, Domian, Start and end of valid time(UTC)
+
+                event.add('time.source', value[2] + ' UTC')
```

Comment:
yes. If we can tell ISPs that we according to the feed this is a valid point in time where their logs (proxy logs, netflow ,...) would find this IP address behaving in this particular way.